src/tools/env.c

```diff
@@ -569,7 +570,7 @@ void RecordEnvMappings(uintptr_t addr, size_t length, int fd)
     int ret;
     mapping_t* mapping = NULL;
```

Comment:
why not testing for mapping_entry here, and don't do the search if it's NULL, so we avoid creating the hash for nothing?

Reply:
Done

Reply:
I was more thinking of putting lines 571-590 behind a `if(mapping_entries) { ... }` but it doesn't matter much actualy.